bundles/org.openhab.binding.automower/src/main/java/org/openhab/binding/automower/internal/things/AutomowerHandler.java

```diff
@@ -94,21 +115,101 @@ public class AutomowerHandler extends BaseThingHandler {
         }
     };
 
-    public AutomowerHandler(Thing thing) {
+    public AutomowerHandler(Thing thing, TimeZoneProvider timeZoneProvider) {
         super(thing);
+        this.timeZoneProvider = timeZoneProvider;
+        this.mowerZoneId = timeZoneProvider.getTimeZone(); // default initializer
     }
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
         if (RefreshType.REFRESH == command) {
             logger.debug("Refreshing channel '{}'", channelUID);
-            refreshChannels(channelUID);
+            // refreshChannels(channelUID); // causes >100 channel updates during setup
```

Comment:
Left by mistake? Log says "Refreshing channel", so something seems not fully right now.

Reply:
removed the logger as well as the commented `refreshChannels` call:

```
        if (RefreshType.REFRESH == command) {
            // not implemented as it would causes >100 channel updates in a row during setup (performance)
        } else {
```